### ISSUE TITLE
Doppler tuning seems to stop when the Input tab is not displayed

### ISSUE THREAD

**Issue by ac2cz** [ac2cz/FoxTelem#347]
I am using 1.05q on the Fox-1e EM stack and PSK.  I left it maybe 1.5hours ago.  Just went to check the web site and found it had stopped reporting packets maybe 45 minutes ago.  Came to look and it was on the health tab for Fox-1e.  When I selected "Input" I could see that the demodulation location was 25 or 30Khz too high.  But just as I started to look at it, the demodulation bars shifted slowly until it found the carrier and locked in.  Now it is running again.

So in other words, it drifted off the frequency when the input window was not being displayed, but apparently when the window was displayed the 'find it' algorithm took over again.  "Track Doppler" is checked, but "Find Signal" is not.

**Comment by ac2cz** [comment]
I played my test file through with real doppler drift from Fox-1A and this worked fine.  I tried it watching the FFT and watching the Health Tab.

I suspect this is a real issue, but its a problem with the frequency tracking for PSK and not for FM-DUV or FSK.  I'm going to move it from the 1.5 release to the 1E Flight milestone.

**Comment by WA4SCA** [comment]
I looked for this on a live AO-85 pass by selecting the Health tab just after AOS.  I waited until well after TCA, when the difference in downlink would be ~5-6 KHz compared to AOS and switched back to Input while looking at the FFT screen as it returned.  There was no jump.

**Comment by burnsfisher** [comment]
Even with PSK it is not always an issue.  I can't figure out just when it happens.

**Comment by burnsfisher** [comment]
It happened 2 more times on 1.05t:  I just got down to look.  The transmitter power looks ok (as seen from the size of the hump in the FFT).  Tuning was just off and had been for 8 hours.  

I think I had the health screen on when I came down.  I switched to Input, saw what I just described, and then after 30-sec or a minute, the demod frequency drifted back, found the signal, and locked on.  So it is clearly related to the input screen actually being displayed.   I'm leaving now but will check the log later and also drop in a screen shot.

**Comment by ac2cz** [comment]
This must be a PSK or 1E issue as we have not seen it for 1A. Burns, please shout if that is not true.  This is assigned to the 1E Flight Release.  If it is impacting testing I can more it to the 1E Test Release, which is my current priority.

**Comment by burnsfisher** [comment]
I think I can work around it be keeping the "input" screen in front.

**Comment by ac2cz** [comment]
I think there were several bugs in the Doppler tuning that could make it look like tuning stopped when the Input tab was not displayed.  Those bugs are fixed in 1.06g

**Comment by burnsfisher** [comment]
1.06h and 1E flight model flat sat:  I stopped the decoder, then turned off the satellite transmission.  Then I clicked on the FFT graph so the tuning was way off to one side of the Fox-1E range.  I then switched to the health screen and started the satellite.  With "Find Signal when peak over 12dB" it locked in quickly and started doing a good job decoding.  With Find Signal set to the default of 14dB it did not lock in.  Closing this one, and I'll file another one suggesting the default should be 12 rather than 14.  (I know that it was 10 originally).
